src/toir.h

```diff
@@ -18,6 +18,6 @@
 elem *incUsageElem(IRState *irs, Loc loc);
 elem *getEthis(Loc loc, IRState *irs, Dsymbol *fd);
 elem *setEthis(Loc loc, IRState *irs, elem *ey, AggregateDeclaration *ad);
-int intrinsic_op(char *name);
+int intrinsic_op(FuncDeclaration *name);
```

Comment:
As `va_start` is templated on the type of the named arg, the mangling will change with the type of the arg.  This means looking at the `FuncDeclaration` is a lot more straightforward than trying to deconstruct the mangled name to determine if we have the correct function.